motoman_driver/MotoPlus/Controller.c

```diff
@@ -791,6 +735,18 @@ BOOL Ros_Controller_StatusUpdate(Controller* controller)
 						}
 						break;
 					}
+					case IO_ROBOTSTATUS_PFL_STOP: // PFL Stop
+					case IO_ROBOTSTATUS_PFL_ESCAPE: //  PFL Escaping
+					{
+						if (ioStatus[i] == 1)  // signal turned ON

```

Comment:
Not necessarily in this PR, but might be nice if we could use the constant here instead of `i`.

Reply:
ok